.circleci/config.yml

```diff
@@ -4,6 +4,9 @@ executors:
   docker-executor:
     docker:
       - image: circleci/android:api-28
+        auth:
+          username: $DOCKERHUB_USERNAME
+          password: $DOCKERHUB_ACCESS_TOKEN
```

Comment:
Can you double-check that this is still true for `amplify-flutter`?
> By default, CircleCI does not pass secrets to builds from forked PRs for open source projects and hides four types of configuration data:
> 1. Environment variables set through the application.
> ...

ref: [CircleCI Documentation, "Pass Secrets to Builds From Forked Pull Requests"](https://circleci.com/docs/2.0/oss/#pass-secrets-to-builds-from-forked-pull-requests)

Reply:
Yea, we'll need to explore how to safely pass this to forks as well (probably using a [Context](https://circleci.com/docs/2.0/contexts/))
It won't be possible to use Docker in CCI unauthenticated starting Nov. 1